database/kangu-product_2016-01-25.sql

```diff
@@ -0,0 +1,70 @@
+# ************************************************************
+# Sequel Pro SQL dump
+# Version 4499
+#
+# http://www.sequelpro.com/
+# https://github.com/sequelpro/sequelpro
+#
+# Host: 127.0.0.1 (MySQL 5.5.38)
+# Database: kangu-product
+# Generation Time: 2016-01-25 10:35:42 +0000
+# ************************************************************
+
+
+/*!40101 SET @OLD_CHARACTER_SET_CLIENT=@@CHARACTER_SET_CLIENT */;
+/*!40101 SET @OLD_CHARACTER_SET_RESULTS=@@CHARACTER_SET_RESULTS */;
+/*!40101 SET @OLD_COLLATION_CONNECTION=@@COLLATION_CONNECTION */;
+/*!40101 SET NAMES utf8 */;
+/*!40014 SET @OLD_FOREIGN_KEY_CHECKS=@@FOREIGN_KEY_CHECKS, FOREIGN_KEY_CHECKS=0 */;
+/*!40101 SET @OLD_SQL_MODE=@@SQL_MODE, SQL_MODE='NO_AUTO_VALUE_ON_ZERO' */;
+/*!40111 SET @OLD_SQL_NOTES=@@SQL_NOTES, SQL_NOTES=0 */;
+
+
+# Dump of table tbl_advert
+# ------------------------------------------------------------
+
+DROP TABLE IF EXISTS `tbl_advert`;
+
+CREATE TABLE `tbl_advert` (
+  `advert_id` int(11) NOT NULL AUTO_INCREMENT,
+  `advert_info` varchar(500) NOT NULL,
```

Comment:
This table still needs an 'advert_user_name' field to store the creator name of the advert. Advert info just shows the personalized advert message.